.github/workflows/pullrequest.yml.template

```diff
@@ -0,0 +1,76 @@
+# Name of the PlaceHolder                 -  Description
+########################################################################################################################
+# <Name of the WorkFlow>                  - Workflow name. example: Bikes
+# <Path to the service>                   - Path to the service where the developer would like to trigger the Flow/ action on pull request - Example: samples/BikeSharingApp/Bikes. 
+# <Path to the Dockerfile>                - Path to the folder which contains the Dockerfile

```

Comment:
Path to the directory that contains the DockerFile for the modified service.

Reply:
Updated.